src/vstest.console/Resources/xlf/Resources.cs.xlf

```diff
@@ -1645,6 +1645,11 @@
         <target state="translated">    Přeskočené: {0}</target>
         <note></note>
       </trans-unit>
+      <trans-unit id="InvalidArgument">
```

Comment:
Please push the xlf file changes, these seem to have been missed

Reply:
re-pushed the files